core/vm/actor/actor.hpp

```diff
@@ -8,28 +8,34 @@
 
 #include <libp2p/multi/content_identifier_codec.hpp>
 
+#include "primitives/address/address.hpp"
 #include "primitives/big_int.hpp"
 
 namespace fc::vm::actor {
-  using primitives::BigInt;
   using libp2p::multi::ContentIdentifier;
+  using primitives::address::Address;
+  using primitives::BigInt;
 
   /** Common actor state interface */
   struct Actor {
-    ContentIdentifier code;
-    ContentIdentifier head;
+    ContentIdentifier code{{}, {}, libp2p::multi::Multihash::create({}, {}).value()};
```

Comment:
consider adding default constructor to ContentIdentifier or a function creating empty ContentIdentifier.